### ISSUE TITLE
Video Assets are saved in Azure storage but preview on PimCore doesn't work

### ISSUE THREAD

**Issue by Dirndorfer** [rohitrajv5/pimcore-bundle-azure-blob-sync#4]
Hi, 
thanks for this greate PimCore extension! 
It works perfect with pictures for me. Video Assets are also saved in Azure storage, but the preview on PimCore doesn't work.
Is this a known Issue and is there a way to fix it?

The Synfony error Log :

ERROR10:57:34 | pimcore | Couldn't create thumbnail of video /Masks/videos/200429_Masken-allgemein.mp4
-- | -- | --
ERROR10:57:34 | pimcore | Exception: Unable to convert video, see logs for details. in /var/www/html/my-project/vendor/pimcore/pimcore/models/Asset/Video/Thumbnail/Processor.php:114 Stack trace: #0 /var/www/html/my-project/vendor/pimcore/pimcore/models/Asset/Video.php(157): Pimcore\Model\Asset\Video\Thumbnail\Processor::process(Object(Pimcore\Model\Asset\Video), Object(Pimcore\Model\Asset\Video\Thumbnail\Config), Array) #1 /var/www/html/my-project/vendor/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/AssetController.php(1548): Pimcore\Model\Asset\Video->getThumbnail(Object(Pimcore\Model\Asset\Video\Thumbnail\Config), Array) #2 /var/www/html/my-project/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(158): Pimcore\Bundle\AdminBundle\Controller\Admin\AssetController->getPreviewVideoAction(Object(Symfony\Component\HttpFoundation\Request)) #3 /var/www/html/my-project/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(80): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #4 /var/www/html/my-project/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php(201): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #5 /var/www/html/my-project/web/app.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request)) #6 {main}

Best Regards
Max

**Comment by cesarab** [comment]
I have the same problem. I took a look at the issues section of the official repo, and I have seen this open issue: https://github.com/pimcore/pimcore/issues/5007

They don't mention this Azure bundle but, being based on the AWS one, I guess they are talking about the same thing.

The problem affects the video preview and the video thumbnail.

Hopefully they will find a solution soon.  There's [a woraround](https://github.com/pimcore/pimcore/issues/5007#issuecomment-755262812) in the comments of that issue (I've tested it and it works, but I'm not sure if it has side effects).

Regards
